File: YesodProjects/Stat/Handler/Write.hs
-- Corresponds to file ..../PHP Files/write.php
module Handler.Write where

import Import
import Data.ByteString.Lazy as B

getWriteR :: Handler Html
getWriteR = do
    liftIO $ B.writeFile "/Users/Nickolas/Documents/write.txt" "The quick brown fox jumps over the lazy dog."
    defaultLayout [whamlet|Written|]

File: PHP Files/write.php
<?php
	// Corresponds to file ../YesodProjects/Stat/Handler/Write.hs
	$myfile = fopen("/Library/WebServer/Documents/write.txt", "w") or die("Unable to open file!");
	echo fwrite($myfile,"The quick brown fox jumps over the lazy dog.");
	fclose($myfile);
?>